starter-code/index.js

```diff
@@ -1,10 +1,47 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function() {
+  this.length=0;
+  this.items=[];
+};
+
+SortedList.prototype.add     = function(item) {
+  if (this.items.length===0) {
+    this.items.push(item);
+  }
+  else {
+    var i=0;
+    while (item>this.items[i]) {
+      i++;
+    }
+    // console.log(this.items, i);
+    this.items.splice(i, 0, item);
```

Comment:
well done ! I think it's might be even better (faster) than the solution we thought of ;)